### PR TITLE
Update a bunch of npm packages and adjust docker-compose and gulpfile

### PR DESCRIPTION
**Pull request by CarstenHoyer**
#### What does this PR do?
This PR updates a few npm packages; gulp-autoprefixer, gulp-notify, gulp-sass, gulp-sourcemaps, jshint, node-sass and their derivatives.

What has then been tested? 

* I can update the scss rules, and that is reflected in the compiled css. 
* Sourcemaps are generated.
* Prefix are being set

Vsuall Tests should be OK. See https://reload.zulipchat.com/#narrow/stream/235451-dev-snak/topic/Backstop.20p.C3.A5.20GitHub.20Actions/near/264128682

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -104,7 +104,7 @@ hooks:
       export NVM_DIR="$HOME/.nvm"
       [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
       nvm current
-      nvm install 9.11.2
+      nvm install 15.14.0
```
Comment (mikkelblyme) [inline .platform.app.yaml:107]: Hvorfor ikke opdatere til LTS versionen? 😄

Comment (CarstenHoyer) [reply .platform.app.yaml:107]: Det var for at køre med den samme som i docker-compose. Men det kan også være det er godt nok med LTS-versionen

Comment (mikkelblyme) [reply .platform.app.yaml:107]: Ja, de skal følges ad. Jeg ser ingen grund til ikke at køre LTS

Comment (CarstenHoyer) [reply .platform.app.yaml]: Fixed

**Pull request merged**